packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -501,9 +506,14 @@ const ContextMenuSubContent = React.forwardRef<
       ref={forwardedRef}
       style={{
         ...props.style,
-        // re-namespace exposed content custom property
-        ['--radix-context-menu-content-transform-origin' as any]:
```

Comment:
This was exposed as `…-context-menu-content-…`, even though it's the sub content. I've done the same for now, but I'm wondering whether it was an omission or on purpose, I can see a reason for both…

Reply:
I remember discussing this and name spacing with `sub` made it harder to share a common block of content styles. I think we ended up leaving it this way for the ergonomics of that.

Reply:
Yeah exactly that's what I was thinking too.
So we're happy with the same thinking for all these new variables representing content/trigger (vs subcontent/subtrigger)?

Reply:
Sub triggers are a bit different as they wouldn't have many common styles but I think this is fine tbh

Reply:
Yeah I think it's ok too because it's more about the relative aspect of styling the content according to "its" trigger.